ipatests/test_webui/test_automember.py

```diff
@@ -285,6 +287,7 @@ def test_rebuild_membership_hosts(self):
         # Rebuild membership for first host, using action on host details facet
         self.navigate_to_record(host1, entity='host')
         self.action_list_action('automember_rebuild')
+        self.assert_notification(type='warning', assert_text=rebuild_warning_msg)
```

Comment:
The message is not printed in a notification but rather in the dialog confirmation box, which has already been closed because the method `action_list_action` called 1 line above has a default argument `confirm=true` (see https://github.com/freeipa/freeipa/blob/8d92546447351bb99a71352fd1e6121cfd8a8c35/ipatests/test_webui/ui_driver.py#L1808).
If you call `action_list_action` with `confirm=False` instead, you should be able to use `self.get_dialog_info()` to access the message inside the dialog. Don't forget to call `self.dialog_button_click` after checking the message, in order to confirm the action.

Reply:
in other words, it would look something like this
```
self.action_list_action('automember_rebuild', confirm=False)
dialog_info = self.get_dialog_info()
# do some assertion with dialog_info['text']
self.dialog_button_click(confirm_btn)  # if no dialog is provided, the active one is selected
```